disable-all-wp-updates.php

```diff
@@ -182,6 +189,76 @@ public function admin_init() {
 		remove_action( 'network_admin_notices', 'maintenance_nag' );
 	}
 
+	/**
+	 * Block users from taking certain actions.
+	 *
+	 * Adds `do_not_allow` magic keyword to block users from taking update
+	 * related actions.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @param string[] $caps Required primitive caps for the capability been checked.
```

Comment:
Is `string[]` correct? Is it not an `array` being passed?

Reply:
The `[]` indicates array of type `string`. I think it's a relatively recent addition to PHP docblock standards so IDE's can catch errors. https://docs.phpdoc.org/3.0/guide/guides/types.html#arrays